scripts/create_ts_for_integration_tests.py

```diff
@@ -176,25 +182,27 @@ def create_dense_time_series(ts_api):
         df_add(pd.DataFrame({NAMES[i + 4]: arr + i + 5}, index=millisec_idx))
         i += 5
 
-    ts_add(
-        TimeSeries(
-            name=NAMES[113],
-            external_id=NAMES[113],
-            is_string=False,
-            metadata={"offset": "n/a", "delta": "uniform random"},
+    if sys.platform != "win32":
+        # Windows does not support the low bound used for the index in create_dense_rand_dist_ts
```

Comment:
Check the comment above. Numpy functions with default `dtype=int` is 32-bit on Windows for some reason (and not on Unix & Mac).